devops/providers/azure-devops/templates/infrastructure/azure-pipelines-app.yml

```diff
@@ -0,0 +1,61 @@
+parameters:
+  environments: []
+  configurationMatrix: []
+
+stages:
+
+- stage: PublishBuildArtifact
+  jobs:
+  - job: Publish
+    displayName: Pull scripts and validate agent readiness
+    steps:
+    - task: Bash@3
+      name: PullScripts
+      displayName: Download Cobalt scripts
+      inputs:
+        targetType: 'inline'
+        script: |
+          # Note: This is intentionally inline because applications should only have
+          # a minimal pipeline definition and Terraform module (for Cobalt) in their repo.
+          mkdir cobaltsrcripts
+          cd cobaltsrcripts
+          git init
+          git remote add origin https://github.com/microsoft/cobalt.git
+          git fetch --force --tags --prune --progress --no-recurse-submodules origin
+          git checkout --progress --force origin/master
```

Comment:
Do we want to peg this to the tag from which they are versioned to?

Reply:
Intentionally deferring this for now to be covered by #286 to allow time to account for constraints.